tools/setup/emoji/emoji_names.py

```diff
@@ -32,8 +34,279 @@
+    "1f1e6-1f1ec": {"canonical_name": "flag_antigua_and_barbuda", "aliases": []},
+    "1f1e6-1f1ee": {"canonical_name": "flag_anguilla", "aliases": []},
+    "1f1e6-1f1f1": {"canonical_name": "flag_albania", "aliases": []},
+    "1f1e6-1f1f2": {"canonical_name": "flag_armenia", "aliases": []},
+    "1f1e6-1f1f4": {"canonical_name": "flag_angola", "aliases": []},
+    "1f1e6-1f1f6": {"canonical_name": "flag_antarctica", "aliases": []},
+    "1f1e6-1f1f7": {"canonical_name": "flag_argentina", "aliases": []},
+    "1f1e6-1f1f8": {"canonical_name": "flag_american_samoa", "aliases": []},
+    "1f1e6-1f1f9": {"canonical_name": "flag_austria", "aliases": []},
+    "1f1e6-1f1fa": {"canonical_name": "flag_australia", "aliases": []},
+    "1f1e6-1f1fc": {"canonical_name": "flag_aruba", "aliases": []},
+    "1f1e6-1f1fd": {"canonical_name": "flag_åland_islands", "aliases": ["flag_aland_islands"]},
+    "1f1e6-1f1ff": {"canonical_name": "flag_azerbaijan", "aliases": []},
+    "1f1e7-1f1e6": {"canonical_name": "flag_bosnia_and_herzegovina", "aliases": []},
+    "1f1e7-1f1e7": {"canonical_name": "flag_barbados", "aliases": []},
+    "1f1e7-1f1e9": {"canonical_name": "flag_bangladesh", "aliases": []},
+    "1f1e7-1f1ea": {"canonical_name": "flag_belgium", "aliases": []},
+    "1f1e7-1f1eb": {"canonical_name": "flag_burkina_faso", "aliases": []},
+    "1f1e7-1f1ec": {"canonical_name": "flag_bulgaria", "aliases": []},
+    "1f1e7-1f1ed": {"canonical_name": "flag_bahrain", "aliases": []},
+    "1f1e7-1f1ee": {"canonical_name": "flag_burundi", "aliases": []},
+    "1f1e7-1f1ef": {"canonical_name": "flag_benin", "aliases": []},
+    "1f1e7-1f1f1": {"canonical_name": "flag_st_barthélemy", "aliases": ["flag_st_barthelemy"]},
+    "1f1e7-1f1f2": {"canonical_name": "flag_bermuda", "aliases": []},
+    "1f1e7-1f1f3": {"canonical_name": "flag_brunei", "aliases": []},
+    "1f1e7-1f1f4": {"canonical_name": "flag_bolivia", "aliases": []},
+    "1f1e7-1f1f6": {"canonical_name": "flag_caribbean_netherlands", "aliases": []},
+    "1f1e7-1f1f7": {"canonical_name": "flag_brazil", "aliases": []},
+    "1f1e7-1f1f8": {"canonical_name": "flag_bahamas", "aliases": []},
+    "1f1e7-1f1f9": {"canonical_name": "flag_bhutan", "aliases": []},
+    "1f1e7-1f1fb": {"canonical_name": "flag_bouvet_island", "aliases": []},
+    "1f1e7-1f1fc": {"canonical_name": "flag_botswana", "aliases": []},
+    "1f1e7-1f1fe": {"canonical_name": "flag_belarus", "aliases": []},
+    "1f1e7-1f1ff": {"canonical_name": "flag_belize", "aliases": []},
+    "1f1e8-1f1e6": {"canonical_name": "flag_canada", "aliases": []},
+    "1f1e8-1f1e8": {"canonical_name": "flag_cocos_(keeling)_islands", "aliases": []},
+    "1f1e8-1f1e9": {"canonical_name": "flag_congo___kinshasa", "aliases": []},
+    "1f1e8-1f1eb": {"canonical_name": "flag_central_african_republic", "aliases": []},
+    "1f1e8-1f1ec": {"canonical_name": "flag_congo___brazzaville", "aliases": []},
```

Comment:
The `___` in these two names would be good to shorten. A line at the end of `cleanup_name`, replacing the regexp `_{2,}` (i.e., a run of two or more `_` characters) with the string `_`, would do it.